forte/integrals/cholesky_integrals.cc

```diff
@@ -143,12 +143,22 @@ double** CholeskyIntegrals::three_integral_pointer() { return ThreeIntegral_->po
 
 ambit::Tensor CholeskyIntegrals::three_integral_block(const std::vector<size_t>& A,
                                                       const std::vector<size_t>& p,
-                                                      const std::vector<size_t>& q) {
-    ambit::Tensor ReturnTensor =
-        ambit::Tensor::build(tensor_type_, "Return", {A.size(), p.size(), q.size()});
-    ReturnTensor.iterate([&](const std::vector<size_t>& i, double& value) {
-        value = three_integral(A[i[0]], p[i[1]], q[i[2]]);
-    });
+                                                      const std::vector<size_t>& q,
+                                                      const std::string& order) {
```

Comment:
If this function is likely to be called only on the C++ side, why not make order an enum?

Reply:
Yes, it is more likely to be called on the C++ side. Let me make it an enum.

Reply:
Changed.